apps/remote_control/lib/lexical/remote_control.ex

```diff
@@ -78,9 +82,18 @@ defmodule Lexical.RemoteControl do
   end
 
   def stop(%Project{} = project) do
-    project
-    |> node_name()
-    |> :slave.stop()
+    node = node_name(project)
+    :ok = :net_kernel.monitor_nodes(true, node_type: :visible)
+    :ok = :rpc.call(node, System, :stop, [])
+
+    receive do
+      {:nodedown, ^node, _} ->
+        :ok
+    after
+      5_000 ->
```

Comment:
this is a long while to wait. 
We could do a gentle `System.stop` followed by a `System.halt` after a specified time.

Reply:
> this is a long while to wait.

How long do you think is better?

Reply:
one second?